db/scheme.sql

```diff
@@ -71,8 +71,11 @@ CREATE TABLE IF NOT EXISTS solana_blocks (
     block_time BIGINT,
     parent_block_slot BIGINT,
     is_finalized BOOL,
-    is_active BOOL
+    is_active BOOL,
+    priority_fee_percentiles INT ARRAY
```

Comment:
Technically you store `compute unit prices` not `priority fees`.

Please rename to:
`cu_price_percentiles`